src/rust-crypto/verification.ts

```diff
@@ -198,8 +214,28 @@ export class RustVerificationRequest
      *
      * @returns Promise which resolves when the event has been sent.
      */
-    public accept(): Promise<void> {
-        throw new Error("not implemented");
+    public async accept(): Promise<void> {
+        if (this.inner.phase() != RustSdkCryptoJs.VerificationRequestPhase.Requested || this._accepting) {
```

Comment:
Just asking in case this is a typo - is this deliberately using `!=`?

Reply:
nope, it's a typo

Reply:
```suggestion
        if (this.inner.phase() !== RustSdkCryptoJs.VerificationRequestPhase.Requested || this._accepting) {
```